css/app-sidebar.scss

```diff
@@ -562,6 +563,9 @@
 
 			textarea {
 				max-height: calc(100vh - 600px);
+
+				// Revert weird margin from global server style sheet and even out margins between properties.
+				margin: 0 0 -2.8px;
```

Comment:
This looks strange to me

Reply:
Yeah, I don't like it either :laughing: This was an evil hack to eliminate the extra margin below a text area but there is a proper way to do it: https://stackoverflow.com/a/7144960